<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>161</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="VerRadio">
     <item>
      <widget class="QPushButton" name="TriangleButton">
       <property name="text">
        <string>triangle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CircleButton">
       <property name="text">
        <string>circle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RhombusButton">
       <property name="text">
        <string>rhombus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SquareButton">
       <property name="text">
        <string>square</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RectButton">
       <property name="text">
        <string>rectangle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Star5Button">
       <property name="text">
        <string>star(5)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Star6Button">
       <property name="text">
        <string>star(6)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Star8Button">
       <property name="text">
        <string>star(8)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HexButton">
       <property name="text">
        <string>hexagon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CrossButton">
       <property name="text">
        <string>cross</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>690</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="VerPS">
     <item>
      <widget class="QLabel" name="SquareLabel">
       <property name="text">
        <string>Square</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PerLabel">
       <property name="text">
        <string>Perimeter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>680</y>
      <width>186</width>
      <height>96</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="VerScroll">
     <item>
      <widget class="QLabel" name="SizeLabel">
       <property name="text">
        <string>               Change size</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="PlussButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="MinusButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="PosLabel">
       <property name="text">
        <string>Change turn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="PosSlider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>760</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="EntYButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>750</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="EnterButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>680</y>
      <width>102</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter positions</string>
    </property>
   </widget>
   <widget class="QLabel" name="EntXButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>710</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ValueLabel2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>740</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UpdateButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>710</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>500</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Characteristics:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UpdateButton_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>710</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>170</width>
      <height>127</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="TypeLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="CurrentPositionLabel">
       <property name="text">
        <string>Curren X and Y:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Current item:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="BackButton">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="NextButton">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="PositionLabelForShow">
       <property name="text">
        <string>(0, 0)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Char1">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>530</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Char3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>650</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Ch1Text">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>550</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Ch2Text">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>610</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Ch3Text">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>670</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Char2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>590</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditX">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>701</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditY">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>740</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="SquareBrowser">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>700</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextBrowser" name="PerimeterBrowser">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>750</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>754</x>
     <y>637</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
